raycicmd/config.go

```diff
@@ -119,6 +119,12 @@ type config struct {
 	// Optional.
 	AllowTriggerStep bool `yaml:"allow_trigger_step"`
 
+	// RestrictedConcurrencyGroup is the list of concurrency group name prefixes that are
+	// allowed for the pipeline. If not set, all concurrency group names are allowed.
+	//
```

Comment:
could you say "if set to `null` or left unset" in the comment here? it is actually possible to explicitly set to `null` here (I think it is `~` or `null`)